.eslintrc.js

```diff
@@ -6,6 +6,7 @@ const config = {
         cy: 'readonly',
         Cypress: 'readonly',
     },
+    ignorePatterns: ['.yarn/**/*.js'],
```

Comment:
We could also use an .eslintignore file here. I don't really have a particular preference though.

Reply:
yeah, good point. That's probably a bit more consistent given the fact we also have a `.prettierignore` now

Reply:
Actually I see we're using a .prettierignore as well. For consistency I'd go with an .eslintignore.

Reply:
le done